lib/shopify_api/resources/order.rb

```diff
@@ -30,6 +30,11 @@ def capture(amount = "", currency: nil)
       Transaction.create(capture_transaction)
     end
 
+    def fulfillment_orders(options = {})
```

Comment:
Could this not be done with a `has_many`?

Reply:
That doesn't quite work for us because `FulfillmentOrder` is both a top-level resource (i.e. /fulfillment_orders/:id) as well as defined within an order (/orders/:order_id/fulfillments). We resolved this by creating the FulfillmentOrder ActiveResource as a top-level one (no :order_id prefix_option) then handling the order.fulfillment_orders with method implementation rather than association.